### PR TITLE
PlayStation.PSPlus version 12.1.0

### PR DESCRIPTION
**Pull request by infinitepower18**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69483)

### CODE REVIEW

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-psplus.playstation.com/downloads/psplus/pc/PlayStationPlus-12.1.0.exe
+  InstallerSha256: 315488B3D09D7F4AC35DE12EAB0BD95782418E222E9A587E7EC0C1C860618693
+  InstallerSwitches:
+    Silent: /quiet
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.installer.yaml:12]:
```suggestion
    Silent: /quiet
    SilentWithProgress: /quiet
```

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+PackageLocale: en-US
+Publisher: Sony Interactive Entertainment Inc.
+PackageName: PlayStation Plus
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml:8]:
```suggestion
PackageName: PlayStationPlus
```
This should match Control Panel Exactly

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+PackageLocale: en-US
+Publisher: Sony Interactive Entertainment Inc.
+PackageName: PlayStation Plus
+License: © 2022 Sony Interactive Entertainment Inc.
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.installer.yaml:14]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+PackageLocale: en-US
+Publisher: Sony Interactive Entertainment Inc.
+PackageName: PlayStation Plus
+License: © 2022 Sony Interactive Entertainment Inc.
+Copyright: Copyright (C) 2022 Sony Interactive Entertainment Inc.
+ShortDescription: PlayStation Plus Game Streaming
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml:13]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (Trenly) [inline manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.yaml:8]:
```suggestion
ManifestVersion: 1.1.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml (hunk 3 above)
Comment (infinitepower18) [reply manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml:9]: How is [this](https://github.com/microsoft/winget-pkgs/blob/master/manifests/p/PlayStation/PSRemotePlay/5.0.0.02220/PlayStation.PSRemotePlay.locale.en-US.yaml) using copyright as license then?

Comment (Trenly) [reply manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml]: Because sometimes things slip past a reviewer

**Review by Trenly — APPROVED** [review]

**Pull request merged**